src/modules/likes.js

```diff
@@ -0,0 +1,29 @@
+const header = new Headers({ 'Content-type': 'application/json; charset=UTF-8' });
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/IvP42xNcmZ7sT5rp87wL/likes/';
+const showData = (data) => {
+  data.forEach((element) => {
+    const id = element.item_id;
+    if (document.getElementById(id)) {
+      document.getElementById(id).innerText = `${element.likes} Likes`;
+    }
+  });
+};
+
+const postLikes = async (button) => {
+  await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: `${button.parentNode.childNodes[1].id}`,
+    }),
+    headers: header,
+  });
+};
+
+const getLikes = async () => {
+  fetch(url)
+    .then((response) => response.json())
+    .then((data) => showData(data));
+};
```

Comment:
**Changes Required**
- [ ] Fair effort on using the `async await`  promise syntactic sugar. However, you're mixing two syntax here, and the `async` keyword in this context is redundant. May we please remove the `.then` and stick to the `async await` format.